src/lib/annotations/BoxAnnotations.js

```diff
@@ -17,6 +18,10 @@ const ANNOTATORS = [
     }
 ];
 
+const ANNOTATOR_TYPE_CONTROLLERS = {
+    [TYPES.draw]: DrawingModeController
```

Comment:
We can do this but the time complexity of access will be linear (linear search to match type) rather than using the type as the key (constant time). This shouldn't be too bad as long as the number of controllers stays small. Let me know if this small time tradeoff is preferable for style.